### PR TITLE
Fix issue 28318: Link to last article

### LINKED ISSUE #28318: "In The Last Article" - Without Adding Any Reference.
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises

### What specific section or headline is this issue about?

Not Added Any Reference to The Previous Page.

### What information was incorrect, unhelpful, or incomplete?

In this content page: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises, A Paragraph starts as In The Last Article. But We don't see any reference to the **last article**. Not all of us are coming to read the reference synchronously. So better having a link back to the previous article is necessary for a good user experience.  

### What did you expect to see?

A Link back to the The Last Article. 

### Do you have any supporting links, references, or citations?

In the 2nd paragraph (https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises)

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/javascript/asynchronous/promises`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/asynchronous/promises/index.md
* Last commit: https://github.com/mdn/content/commit/acfe8c9f1f4145f77653a2bc64a9744b001358dc
* Document last modified: 2023-07-07T07:19:19.000Z

</details>
```[tasklist]
### Tasks
```

### PR DESCRIPTION
**Pull request by wbamberg**
Fixes https://github.com/mdn/content/issues/28318.

### CODE REVIEW

File: files/en-us/learn/javascript/asynchronous/promises/index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ page-type: learn-module-chapter
   </tbody>
 </table>
 
-In the last article, we talked about the use of callbacks to implement asynchronous functions. With that design, you call the asynchronous function, passing in your callback function. The function returns immediately and calls your callback when the operation is finished.
+[In the last article](/en-US/docs/Learn/JavaScript/Asynchronous/Introducing), we talked about the use of callbacks to implement asynchronous functions. With that design, you call the asynchronous function, passing in your callback function. The function returns immediately and calls your callback when the operation is finished.
```
Comment (dipikabh) [inline files/en-us/learn/javascript/asynchronous/promises/index.md:27]: we can change the link text
```suggestion
In the [previous article](/en-US/docs/Learn/JavaScript/Asynchronous/Introducing), we talked about the use of callbacks to implement asynchronous functions. With that design, you call the asynchronous function, passing in your callback function. The function returns immediately and calls your callback when the operation is finished.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks, Will!

File: files/en-us/learn/javascript/asynchronous/promises/index.md (hunk 1 above)
Comment (wbamberg) [reply files/en-us/learn/javascript/asynchronous/promises/index.md]: Yes, better :).

**Review by dipikabh — APPROVED** [review]
👍

**Pull request merged**